8.cytopick_analysis/scripts/prediction_images_find.py

```diff
@@ -508,20 +568,20 @@ def random_cell_select(
                 pathlib.Path(
                     f"{image_out_dir_path}/{key}_{channel1}_{channel2}_{channel3}_composite_image_crop_cell_{cell}.tiff"
                 ),
-                im_crop,
+                composite_image_crop,
                 compression=None,
             )
             df = df.to_frame().T
             df[
                 "image_path"
-            ] = f"{image_out_dir_path}/{key}_{channel1}_{channel2}_{channel3}_composite_image_crop_cell_{cell}.tiff"
+            ] = f"{image_out_dir_path}/{key}_{channel1}_{channel2}_{channel3}_composite_image_cell_{cell}.png"
```

Comment:
What incurred the switch from `tiff` to `png`? Would it be worth a quick comment here to describe why? Is there any data or provenance loss as a result of this?

Reply:
PNGs saved for figure assembly. I am not worried about data loss as I am not overwriting the files and they are for visualization purposes only.